src/Compilers/VisualBasic/Portable/Lowering/LambdaRewriter/LambdaRewriter.vb

```diff
@@ -681,7 +694,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Dim frame As LambdaFrame = Nothing
```

Comment:
It is to avoid warning saying "passed by reference before it has been assigned a value" when calling TryGetValue(node, frame).